x/group/client/cli/tx_test.go

```diff
@@ -0,0 +1,1844 @@
+package cli_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+
+	"strings"
```

Comment:
nit: the formatting is off here. Imports should always be

(
  stdlib

  misc

  cosmos-sdk
)

If you're using VSCode, or a similar IDE, it should do this automatically for you.

If you are using VSCode, make sure to have the following added to your sdk workspace config:

```json
  "gopls": {
    "local": "github.com/cosmos/cosmos-sdk"
  }
```